.test/example-commands.sh

```diff
@@ -0,0 +1,27 @@
+# docker:24.0.7-cli [amd64]
+# <pull>
+
+# </pull>
```

Comment:
I could work harder to collapse this, but :shrug: (I didn't think it was worth it -- happy to adjust if you think it's worth doing, @yosifkit).

Reply:
It seems fine.